style.css

```diff
@@ -0,0 +1,143 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700&display=swap");
+
+body {
+  font-family: 'Poppins'sans-serif;
```

Comment:
[OPTIONAL]
- [ ] The font `Poppins` in not applying to your texts, you can fix it by change this line to this code
`font-family: 'Poppins', sans-serif;`